speedrun-3.html

```diff
@@ -95,8 +95,8 @@ <h3>Made by Ben Songster (MeemawHustlin) with help from the PRCLive community.</
 <ol>
 <li>Click continue on the quest pop-up to start the run. Press caps lock to always be sprinting, and press <kbd>F3</kbd> to bring up the spell menu.</li>
 <li>Hotkey <span class="spellname">Flare</span> and equip <span class="spellname">Adrenaline Rush</span>.</li>
-<li>Cast <span class="spellname">Adrenaline Rush</span> and <a href="./glitches.html#saveclip" target="_blank">Save Clip</a><a class="screenshot" href="images/saveclips/SC_Tutorial_1.webp" target="_blank"></a> through the right front edge of the NE archway.</li>
-<li>When you clip through the wall, run to the east until you spawn back in bounds. This is called <a href="./glitches.html#voidwarp" target="_blank">Void Warping</a>. If the clip doesn’t work, try setting the quicksave up again and loading again.</li>
+<li>Cast <span class="spellname">Adrenaline Rush</span> and <span class="glitch">Save Clip</span><a class="screenshot" href="images/saveclips/SC_Tutorial_1.webp" target="_blank"></a> through the right front edge of the NE archway.</li>
+<li>When you clip through the wall, run to the east until you spawn back in bounds. This is called <span class="glitch" clid="glitchvoidwarp">Void Warping</span>. If the clip doesn’t work, try setting the quicksave up again and loading again.</li>
```

Comment:
to clarify, there's no easy way to get this glitch class to accept a list of variations on the term and get it to link to the same place? I have to semi-manually swap 316 instances of glitch links to be these spans in the v4 guide so I want to make absolutely sure the glitch class isn't going to change later before doing that.

Reply:
you could make multiple "glitches" that all link to the same place, but I think clid is easy enough. There were only like 10 that didn't follow the standard title.